examples/notebooks/segmentation-tutorial.ipynb

```diff
@@ -827,7 +829,7 @@
    "outputs": [],
    "source": [
     "if is_alchemy_used:\n",
-    "    from catalyst.dl.runner import SupervisedAlchemyRunner as SupervisedRunner\n",
+    "    from catalyst.contrib.runner.alchemy import SupervisedAlchemyRunner as SupervisedRunner\n",
```

Comment:
you don't need this, cause dl import everything from contrib already

Reply:
no, the current version in master throws an error. u can try it by yourself

Reply:
not reproduced